index.html

```diff
@@ -5,23 +5,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Awesome books</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+ <main>
     <div class="awesome">
-    <h1>Awesome Books</h1>
-        <div id="book-list"></div><br>
+    <h1>All awesome books</h1>
+    <table>
+        <thead>
+            <tr>
+            <th>Title</th>
+            <th>Author</th>
+            <th></th>
+            </tr>
+        </thead>
+        <tbody id="book-list"></tbody>
```

Comment:
Please follow the wireframe for the books list.
Yours:
![Yours](https://user-images.githubusercontent.com/35504149/143427823-71032cb0-d486-49ae-90d7-46770a65d217.png)
Wireframe:
![o](https://user-images.githubusercontent.com/35504149/143427849-4aa92e2c-8e12-4ba2-a8ef-2ea96350dba7.png)